glowbot/commands.py

```diff
@@ -155,7 +164,9 @@ async def claim(self, ctx: discord.ApplicationContext, hours: Option(
                     await ctx.respond(f'{ctx.author.mention}: ❌ Sorry, not enough banked time to claim `{hours}` hour(s) of VIP (Currently have `{player.seeding_time_balance.seconds // 3600}` banked hours).', ephemeral=True)
                     return
                 else:
-                    player.seeding_time_balance -= timedelta(hours=hours)
+                    # policy question... should ppl accrue seeding hours if they already have
+                    # permanent vip? i.e., they currently purchase vip or have free vip through
+                    # some other means?
```

Comment:
Pure FYI; you can ask these questions directly in the PR thread as well and link to code lines in github, and they'll be recorded, no need to put them in comments (I greatly appreciate you checking) 😄

Policy decision on this is for now "yes"; even if you have VIP/indefinite VIP/are paying for VIP, you can still accrue time into banked and total seeding times.  I'm open to PR's that allow this to be configurable but in lieu of that the default should still be to accrue IMO.

Reply:
Thanks.

I agree with the accrual. It was just something I thought of when going through the code.